app/components/agenda/agenda-header/agenda-tabs.hbs

```diff
@@ -1,11 +1,7 @@
 <Auk::Tabs @reversed={{true}}>
-  <Auk::Tab @route="agendas.overview" @isHierarchicalBack={{true}} />
+  <Auk::Tab @route="agendas" @isHierarchicalBack={{true}} />
   <Auk::Tab
     @route="agenda.agendaitems.index"
-    @query={{if
-      this.currentAgendaItemId
-      (hash anchor=this.currentAgendaItemId)
-    }}
```

Comment:
Waarom is deze query param verwijderd? Die zorgt ervoor dat in het detail automatisch naar het juiste agendapunt gescrolled wordt afaik.

Reply:
De `should test scroll anchor` test van `agendaitem-changes.spec.js` faalde op [deze lijn](https://github.com/kanselarij-vlaanderen/frontend-kaleidos/blob/development/cypress/integration/unit/agendaitem-changes.spec.js#L274). Probleem zijnde dat als in "Overzicht" bent, da op de "Details" klikt, dan terug "Overzicht" de originele anchor verandert, waardoor er niet gescrolld werd naar het juiste agendapunt.

Hoe ik de error begreep:

Op [deze lijn in `agenda-tabs.js`](https://github.com/kanselarij-vlaanderen/frontend-kaleidos/blob/development/app/components/agenda/agenda-header/agenda-tabs.js#L33) krijgen we de modellen die aan de route van de "Details" tab gegeven worden. Vanuit "Overzicht" gaat `currentAgendaItemId` null zijn, want we zitten daar in de `agenda.agendaitems.index` route, niet in een subroute van `agenda.agendaitems.agendaitem`. Dus de route voor de "Details" tab zal naar het agendapunt `firstAgendaitem` gaan. Eens daar wordt de route voor de "Overzicht" opnieuw berekend, en deze keer is er wel een `currentAgendaItemId`, nl. de `firstAgendaitem`. Dus de anchor wordt overschreven met een foute/onverwachte waarde.

Na nader inzien is dit wel geen echte fix, want als er van agendapunt geswitcht wordt in de "Details" view, dan wordt die anchor niet juist gezet